src/site/content/en/blog/aspect-ratio/index.md

```diff
@@ -0,0 +1,211 @@
+
+Now that we have identified the aspect ratio value, we can apply that to our
+parent container. Consider the following example:
+
+```html
+<div class="container">
+  <img class="media" src="..." alt="">
+</div>
+```
+
+We could then write the following CSS:
+
+```css
+.container {
+  position: relative;
+  width: 100%;
+  padding-top: 56.25%; /* 16:9 Aspect Ratio */
+}
+
+.media {
+  position: absolute;
+  top: 0;
+}
+```
+
+## Maintaining aspect ratio with `aspect-ratio`
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="aspectratio-future.jpg" alt="">
+  <figcaption class="w-figcaption">
+    Using `aspect-ratio` to set a 1:1 aspect ratio on post preview images within a carousel.
+  </figcaption>
+</figure>
+
+Unfortunately, calculating these `padding-top` values is not very intuitive, and
+requires some additional overhead and positioning. With the new intrinsic
+`aspect-ratio` [CSS property](https://www.w3.org/TR/css-sizing-4/#aspect-ratio),
+coming to Chromium 88, Safari Technical Preview 118, and Firefox __, the
+language for maintaining aspect ratios is much more clear.
+
+With the same markup, we can replace: `padding-top: 56.25%` with `aspect-ratio:
+16 / 9`, setting `aspect-ratio` to a specified ratio of `width` / `height`.
+
+<div class="w-columns">
+{% Compare 'worse', 'Using padding-top' %}
+```css
+.container {
+  width: 100%;
+  padding-top: 56.25%;
+}
+```
+{% endCompare %}
+
+{% Compare 'better', 'Using aspect-ratio' %}
+```css
+.container {
+  width: 100%;
+  aspect-ratio: 16 / 9;
+}
+```
+{% endCompare %}
+</div>
+
+Using `aspect-ratio` instead of `padding-top` is much more clear, and does not
+overhaul the padding property to do something outside of its usual scope.
+
+Now, not only is the aspect ratio much more clear, but this new property also
+adds the ability to set aspect ratio to `auto`, where “replaced elements with an
+intrinsic aspect ratio use that aspect ratio; otherwise the box has no preferred
+aspect ratio." If both `auto` and a `<ratio>` are specified together, the
+preferred aspect ratio is the specified ratio of `width` / `height` unless it is
+a replaced element with an intrinsic aspect ratio, in which case that aspect
+ratio is used instead.
+
+This works really well with CSS layout mechanisms like CSS Grid and Flexbox as
+well. Consider a list with children that you want to maintain a 1:1 aspect
+ratio, such as a grid of sponsor icons:
+
+```html
+<ul class="sponsor-grid">
+  <li class="sponsor">…
+  <li class="sponsor">…
+</ul>
+```
+
+```css
+.sponsor-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(120px, 1fr));
+}
+
+.sponsor img {
+  aspect-ratio: 1/1;
+  width: 100%;
+  object-fit: contain;
+}
+```
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="sponsors.gif" alt="">
+  <figcaption class="w-figcaption">
+    [Demo](https://codepen.io/una/pen/e89deff424a107349a0f1e30908cddbf?editors=0110) _TODO: make public_ and make this a hosted video
```

Comment:
More Markdown in HTML (I recognize that this is placeholder content)

![image](https://user-images.githubusercontent.com/4713486/104795191-4a01ac80-5761-11eb-84be-077e77e9826d.png)